src/packages.builds

```diff
@@ -10,7 +16,7 @@
     <Project Include="Native\pkg\**\*.builds">
       <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true</AdditionalProperties>
     </Project>
-    <Project Include="*\pkg\*.builds">
+    <Project Include="*\pkg\*.builds" Condition="'$(FilterToOSGroup)' == 'Windows_NT'">
```

Comment:
So this means that packages will only be produced if you are targeting Windows_NT, right? Are we saying that you can't build packages on other OS's then?